tldw_Server_API/app/services/admin_usage_service.py

```
@@ -1012,3 +1012,65 @@ def _fmt(value):
+            }
+            # Estimated cost at blended $3/M tokens
+            tokens = int(r.get("total_tokens") or 0)
+            r["estimated_cost_usd"] = round(tokens * 0.000003, 4)
+            items.append(r)
+
+        return {
+            "group_by": group_by,
+            "range_days": range_days,
+            "items": items,
+        }
+    except Exception as exc:
```

Ruff (0.15.7)

[warning] Do not catch blind exception: `Exception`

(BLE001)